assets/js/barChart.js

```diff
@@ -69,11 +69,25 @@ class BarChart {
             .enter().append('g');
 
         this.rects = bars.append('rect')
-            .attr('y', function(d) { return yScale(d.name); })
-            .attr('height', yScale.rangeBand());
+            .attr('y', function(d) { return yScale(d.trimName); })
+            .attr('height', yScale.rangeBand())
+            .on("mouseover", function (d) { tooltip.style("display", "inline-block"); })
```

Comment:
We should also have a tooltip on the y-axis label. I've noticed some cases where the bar is so small that there's actually no way to trigger the tooltip.